src/main/groovy/edu/oregonstate/mist/locations/core/ArcGisLocation.groovy

```diff
@@ -17,7 +17,22 @@ class ArcGisLocation extends BaseType {
     String latitude
     @JsonProperty("Longitude")
     String longitude
-    Double[][][] coordinates
+    def coordinates
+    String coordinatesType
+    @JsonProperty("Count_")
+    Integer giRestroomCount
+    @JsonProperty("Limits")
+    String giRestroomLimit
+    @JsonProperty("LocaAll")
+    String giRestroomLocations
+
+    Integer getGiRestroomCount() {
+        giRestroomCount ?: 0
```

Comment:
In this method, let's also do a validation and check that the count provided is equal to the number of entries in giRestroomLocations . For example, when doing a visual check, I found that Resr provides a count of 5, but 13 are mentioned in the restroom locations